elizabeth/homework1.js

```diff
@@ -0,0 +1,173 @@
+/* Section 1
```

Comment:
Nice work talking through your reasoning. Take a look at the comments below and the homework solutions for some more specific reasoning on why some of these questions work out the way they do. Additionally, don't forget to experiment with different possible values of x!